src/legacy/blueprint-parser.coffee

```diff
@@ -120,5 +126,4 @@ getAstSync = (code) ->
 module.exports = {
   parseApiaryBlueprintSync: getAstSync
   parse: getLocalAst
-  newVersionRegExp: NEW_VERSION_REGEXP # layout control control (editor)
```

Comment:
No reason. Should be encapsulated, IMHO. We should use DC in Apiary if we need this.